mautrix/util/program.py

```diff
@@ -188,6 +188,7 @@ def start_prometheus(self) -> None:
         try:
             enabled = self.config["metrics.enabled"]
             listen_port = self.config["metrics.listen_port"]
+            addr = self.config.get("metrics.addr", "0.0.0.0")
```

Comment:
Since we are using these values here, I think we need to have corresponding `copy`s in `mautrix/bridge/config.py`. See https://github.com/mautrix/twitter/blob/master/mautrix_twitter/config.py#L48-L49 for an example of what it should look like.

Reply:
oh right, didn't notice because I think I don't use that part.
Will update accordingly later:)

Reply:
@sumnerevans OK, I confused a few things when I read your comment, apologies!
I already did that for the bridge I've been using, namely https://github.com/mautrix/telegram/pull/955. In fact, I'm already using that productively, so this code is not untested :) 
Do you want me to file fixup PRs for the other python bridges or is it OK for you to update when bumping mautrix/python there? If so, are all bridges to update in this org?

Reply:
ping @sumnerevans

Reply:
Not sure what should be done here, @tulir ?

Reply:
ping @tulir